samples/openthread/coprocessor/snippets/hci/rcp_hci.conf

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_RCP_SAMPLE_HCI=y
+
+CONFIG_CONSOLE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_HCI_RAW_H4=y
+CONFIG_BT_HCI_RAW_H4_ENABLE=y
+CONFIG_BT_BUF_ACL_RX_SIZE=251
+CONFIG_BT_BUF_EVT_DISCARDABLE_SIZE=255
+
+# Workaround: Unable to allocate command buffer when using K_NO_WAIT since
+# Host number of completed commands does not follow normal flow control.
+CONFIG_BT_BUF_CMD_TX_COUNT=10
+
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor ASA"
+CONFIG_USB_DEVICE_PRODUCT="Thread Co-Processor with BLE HCI"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x0000
```

Comment:
We might want to assign a PID from the reserved list. Search for "Released Products IDs" in the wiki.

Reply:
Can You point me to the wiki? it seems that I can't find it.

Reply:
Sure, done offline.

Reply:
I have reserved CAFD for this one.